frameworks/crewai/email-auto-responder/agents/EmailAutoResponder/config/tasks.yaml

```
@@ -0,0 +1,71 @@
+    A comprehensive email analysis report containing:
+    - Urgency classification with justification
+    - Category assignment with reasoning
+    - Sender assessment
+    - Recommended action plan
+    - Risk assessment if applicable
+
+action_required_emails_task:
+  description: >
+    Based on the email classification, determine the specific action required for this email.
+    Consider the analysis from the filter task and decide:
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)